.travis.yml

```diff
@@ -100,6 +109,7 @@ before_script:
   # urdfdom_headers
   - git clone https://github.com/ros/urdfdom_headers
   - cd urdfdom_headers
+  - git checkout e7e0972a4617b8a5df7a274ea3ba3b92e3895a35
```

Comment:
A small comment about why this commit would go a long way. Thanks.

to quote...
"To fix compilation, we will switch the dependencies to use the commits used in the latest successful model generation"